hack/conversion/conversion.go

```diff
@@ -55,7 +55,7 @@ func main() {
 	generator.OverwritePackage(*version, "")
 	generator.OverwritePackage("api", "newer")
 	for _, knownType := range kapi.Scheme.KnownTypes(*version) {
-		if strings.Contains(knownType.PkgPath(), "GoogleCloudPlatform/kubernetes") {
+		if strings.Contains(knownType.PkgPath(), "k8s.io/kubernetes") {
```

Comment:
Especially that it looks very similarly to `cmd/genconversion/conversion.go` but with less functionality.